core/src/main/scala/neotypes/implicits.scala

```diff
@@ -418,8 +418,10 @@ object implicits {
       new Driver[F](driver)
   }
 
-  implicit def String2QueryBuilder(query: String): DeferredQueryBuilder =
```

Comment:
I realized that having an implicit conversion was a really bad idea.
So, I just replaced both uses of it with _"safer"_ alternatives.